### PR TITLE
Add RN and doc updates for "Wait for timer" renamed to "Timer" in 10.14.0

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-timer.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
 ---
-title: "Wait for Timer"
-url: /refguide/wait-for-timer/
```
Comment (timvandervoornmx) [inline content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-timer.md:3]: Curious what happens here: would this old url redirect to the new one or will it just 404 or something?

Comment (timvandervoornmx) [reply content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-timer.md:3]: Or is that where the alias is for?

**Review by timvandervoornmx — COMMENTED** [review]
Looks good to me, left 1 small question

File: content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-timer.md (hunk 1 above)
Comment (Yiyun333) [reply content/en/docs/refguide/modeling/application-logic/workflows/workflow-elements/wait-for-timer.md:3]: The old url will get redirected to the new one ;)

**Comment by Yiyun333** [comment]
> Looks good to me, left 1 small question

Great. I'll merge the PR by the end of the day.

**Pull request merged**